src/mokka/src/main/java/pl/hycom/mokka/service/file/FileService.java

```diff
@@ -0,0 +1,11 @@
+package pl.hycom.mokka.service.file;
+
+import java.io.File;
+
+/**
+ * @author Mariusz Krysztofowicz (mariusz.krysztofowicz@hycom.pl)
+ */
+@FunctionalInterface
```

Comment:
Functionallnterface is useful only when we really want to make interface functional. I do not think it is case here.